modules/govuk/manifests/apps/search_api.pp

```diff
@@ -206,4 +206,9 @@
       value   => $oauth_secret;
   }
 
+  # Start Elasticsearch in docker if using dev VM
+  if $::govuk_node_class =~ /^development$/ {
+    include govuk_search::docker_elasticsearch
```

Comment:
Should this instead be in the node manifest for the dev VM?

Reply:
Yes, of course.  Don't know why I left it in here :/.